.github/workflows/update_appveyor_image.yml

```diff
@@ -23,6 +23,7 @@ jobs:
       with:
         fetch-depth: 0
         token: ${{ steps.generate-token.outputs.token }}
+        ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
whats this for?

Reply:
@MarshallOfSound this is so GitHub actions doesn't checkout a merge commit: https://github.com/actions/checkout?tab=readme-ov-file#checkout-pull-request-head-commit-instead-of-merge-commit